nx.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "./node_modules/nx/schemas/nx-schema.json",
+  "npmScope": "eslint-getluko-mobile",
+  "affected": {
+    "defaultBase": "main"
+  },
+  "implicitDependencies": {
+    "package.json": {
+      "dependencies": "*",
+      "devDependencies": "*"
+    },
+    ".eslintrc.json": "*"
+  },
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "@nrwl/nx-cloud",
+      "options": {
+        "cacheableOperations": [
+          "build",
+          "lint",
+          "test",
+          "e2e"
+        ],
+        "accessToken": "NTg2MGE1NTYtNjBkOS00ZTU1LWFkYWEtNWJhOTZjYzJiODcxfHJlYWQtd3JpdGU="
```

Comment:
I guess this should not be visible in the repository?  🙈

Do we need @nrwl/nx-cloud in this project?

Reply:
Nope finally I managed to use the default runner.